components/left-bar.vue

```diff
@@ -0,0 +1,174 @@
+                <div>Color, Opacity, Width</div>
+                <div></div>
+              </v-card-text>
+            </v-card>
+          </span>
+        </v-tooltip>
+
+        <v-tooltip right class="tooltip-btn">
+          <template v-slot:activator="{ on, attrs }">
+            <v-btn tile class="mt-5" text :color="activateBtn2(3)" v-bind:class="{ activateBtn: isActivate(3) }" @click="clickMenu(3)" v-bind="attrs" v-on="on">
+              <v-icon>fas fa-crop</v-icon>
+            </v-btn>
+          </template>
+          <span>
+            <v-card class="m-0" max-width="400">
+              <v-img class="white--text align-end" height="200px" 
+                src="./assets/ChangeTR.png">
+                <v-card-title>Change Transform</v-card-title>
+              </v-img>
+              <v-card-subtitle class="pb-0">Change 5 properties</v-card-subtitle>
+              <v-card-text class="text--primary">
+                <div>Anchor, Position,</div>
+                <div>Scale, Rotation, Opacity</div>
+              </v-card-text>
+            </v-card>
+          </span>
+        </v-tooltip>
+
+        <v-tooltip right class="tooltip-btn">
+          <template v-slot:activator="{ on, attrs }">
+            <v-btn tile class="mt-5" v-bind:class="{ activateBtn: isActivate(4) }" text :color="activateBtn2(4)" @click="clickMenu(4)" v-bind="attrs" v-on="on">
+              <v-icon>fas fa-palette</v-icon>
+            </v-btn>
+          </template>
+          <span>                  
+            <v-card class="m-0 p-0" max-width="400">
+              <v-img class="white--text align-end" height="200px" 
+              src="./assets/ChangeBG.png">
+                <v-card-title>Top 10 Australian beaches</v-card-title>
+              </v-img>
+              <v-card-subtitle class="pb-0">Number 10</v-card-subtitle>
+              <v-card-text class="text--primary">
+                <div>Whitehaven Beach</div>
+                <div>Whitsunday Island, Whitsunday Islands</div>
+              </v-card-text>
+            </v-card>
+          </span>
+        </v-tooltip>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+module.exports = {
+  name:"left-bar",
+  props:{
+    selectedmenu: Number,
+  },
+  methods:{
+    clickMenu(i){
+      this.$emit('menuclicked', i)
+    },
+    isActivate: function(i) {
+      if (this.selectedmenu === i){
+        return true
+      }else{
+        return false
+      }
+    }
+  },
+  computed:{
+    activateBtn2(){
+      return (i) => {
+        if (this.selectedmenu == i){
+          return 'white'
+        }
+        else{
+          return 'grey'
+        }
+      }
+    }
+  },
+  mounted() {
+    var self = this;
+    self.$emit('child-event', this.$refs.leftBar.clientHeight);
+    window.addEventListener("resize", function(e) {
+      self.$emit('child-event', e.target.innerHeight);
+    });
+  },
+  beforeDestroy() {
+    // window.removeEventListener("resize");
```

Comment:
Please remove unnecessary line.

Reply:
Thank you. but you are currently checking the oldest commit history. Please check it as new.

Reply:
Should we make new pull request for checking only most recent commit?